packages/salesforcedx-vscode/CHANGELOG.md

```diff
@@ -1,3 +1,31 @@
+# 49.12.0 - September 30, 2020
+
+## Added
+
+#### salesforcedx-vscode-core
+
+- Show progress bar when installing dependencies for Function Create ([PR #2563](https://github.com/forcedotcom/salesforcedx-vscode/pull/2563))
+
+- Add feature flag for Functions ([PR #2572](https://github.com/forcedotcom/salesforcedx-vscode/pull/2572))
+
+- Show message if no default org is set when starting Function ([PR #2560](https://github.com/forcedotcom/salesforcedx-vscode/pull/2560))
+
+#### salesforcedx-vscode-soql
+
+- Add textmate grammar for SOQL files ([PR #2562](https://github.com/forcedotcom/salesforcedx-vscode/pull/2562))
```

Comment:
TextMate grammar

Reply:
Just like with functions, we need to remove everything related to vscode-soql until it gets published